src/components/questions.tsx

```diff
@@ -111,7 +111,7 @@ function ParamChanges(props: { changes: Record<string, unknown> }) {
         decimalPlaces,
         decimalPlaces
       );
-      return <>{numeral} IST</>;
+      return <>{new Intl.NumberFormat().format(Number(numeral))} IST</>;
```

Comment:
a little funny to stringify the number, then make it back into a number to make a string. I'd have suggested `stringifyValue` take on the formatting concern.

Reply:
oh but I see that's coming from the SDK so much harder to change. Some impetus for moving out `ui-components` https://github.com/Agoric/agoric-sdk/issues/6591